lib/simulator-management.js

```diff
@@ -36,15 +36,22 @@ async function createSim(caps) {
   if (!deviceName) {
     let deviceNames = 'none';
     try {
-      deviceNames = await simctl
-        .getDevices(platformVersion, platform)
+      deviceNames = (await simctl
```

Comment:
this is a nice catch, which would be hardly possible without proper typing.
I am currently working on appium-ios-simulator lib changes, so it could also be properly typed